manifests/init.pp

```diff
@@ -29,12 +29,37 @@
 ) {
 
   if $enable_reports {
+    if $reports != '' {
+      $reports_setting = $reports
```

Comment:
Should we add a deprecation warning here re: the `$reports` parameter? @bmjen

Reply:
yeah, let's go ahead and add the warning.